### DISCUSSION TITLE
Is some other scaling possible than 2x?

### DISCUSSION THREAD

**Discussion by murlakatamenka** [tkashkin/Adwaita-for-Steam#52] [Ideas]
Linux Steam client has a well-known issue for fractional scaling, see https://github.com/ValveSoftware/steam-for-linux/issues/5908

Can this skin somehow remedy it? Something like 1.5x for those who find default font too small with 1x but everything is too big with 2x.

---

Also @tkashkin I think it's worth being more specific what OS scaling means in the README (`GDK_SCALE=2`), not everyone knows:

> **High resolutions**: all new graphics assets have @2x variants that will be used when scaling is enabled in both Steam and OS settings.

**Comment by tkashkin** [comment]
1.5x or any other scaling is technically possible but it would require all graphics to be redrawn at the required scale and all sizes to be scaled everywhere.

That's a lot of work to support just one custom scale. It would almost be a new skin at this point.

**Comment by murlakatamenka** [comment]
Yeah, I understand. Custom scaling gotta be supported upstream in the first place, it work on Windows but alas not (yet) on Linux.

**Comment by tkashkin** [comment]
Current Steam client should support fractional scaling with:
* `-forcedesktopscaling 2.0` command line argument ([since May 5th](https://steamcommunity.com/groups/SteamClientBeta/announcements/detail/3705943193607193986))
* `STEAM_FORCE_DESKTOPUI_SCALING=2.0` environment variable ([since June 15th](https://store.steampowered.com/news/group/4397053/view/6627657351330751105))

Steam should also try to read scaling options from system settings on some DEs.
